### ISSUE TITLE
Set working directory for running

### ISSUE THREAD

**Issue by TheSunCat** [Civitasv/cmake-tools.nvim#34]
**Describe the problem or limitation you are having**
I am working on a C++ project which accesses files in a `Resources` folder as if they were in the working directory. When running the project through `CMakeRun`, the working directory is that of nvim, so it cannot load the files it needs.

**Describe the solution you'd like**
The ability to configure the working directory used when launching the program.

**Describe alternatives you've considered**
I found [this](https://github.com/microsoft/vscode-cmake-tools/issues/1395#issuecomment-694968338) issue in vscode-cmake-tools, but setting `cwd` in the `require("cmake-tools").setup` call did not appear to do anything. I might be doing this wrong, however, as I'm not sure how `launch.json` settings translate to this project.

**Comment by Civitasv** [comment]
Can you give a specific example? like the file structure of your project

**Comment by TheSunCat** [comment]
Yes of course. I am using [this](https://github.com/samynk/GameDevEngine2) project by adding `DefineGameProject(Test)` at the end of CMakeLists.txt, running cmake once to generate it, then running `nvim Projects/Test/Test.cpp`. The workring directory would ideally be `$(pwd)/Projects/$project/Resources`, so each project can load its own resources, but it appears to be set to the path nvim was invoked from.

**Comment by rohit-kumar-j** [comment]
@Civitasv, I faced a similar limitation. My executable was in `build/bin/Debug/exec.exe`, and it is supposed to produce a `data.txt` file.

When `CMakeRun` is invoked, it produces a file in the directory from which it was run and produces outputs in the same directory (`root_dir/data.txt`) which is undesirable.(`root_dir ~$ build/bin/Deubg/exec.exe`)

On VSCode, when `CMakeRun` is invoked, the executable is run from its own build directory by default: (`root_dir/build/bin/Deubg ~$ exec.exe`) thereby not cluttering the root_dir. The output files are in (`root_dir/build/bin/Debug/data.txt`)

I think a conditional default option to set this behaviour would be a good idea:..

```lua

-- const.lua

const = {
    ...
    cmake_launch_from_build_dir = true,
    ...
}
```

**Comment by rohit-kumar-j** [comment]
@TheSunCat, commit : [a23d98](https://github.com/Civitasv/cmake-tools.nvim/pull/47/commits/a23d98c356361be6f4bcf28c8098f050cecf8498) from pull #47 fixes this launch from build directory. Since there are a few more proposed upgrades in #47, it might take sometime to get tested on linux and MacOS (currently I am testing on windows with MSBuild and MSVC).

Added default option: 
`cmake_launch_from_built_binary_directory = true,`

**Comment by rohit-kumar-j** [comment]
@TheSunCat, #51 does this by default. You can test this.
